autoload/maktaba/ui/selector.vim

```diff
@@ -157,21 +157,21 @@ endfunction
 " Each entry in {infolist} may be either a line to display, or a 2-item list
 " containing `[LINE, DATA]`. If present, DATA will be passed to the action
 " function as a second argument.
-function! maktaba#selector#Create(infolist) abort
+function! maktaba#ui#selector#Create(infolist) abort
```

Comment:
Isn't this a breaking change for anyone using this API already? (Looking at #208, we released this function in Maktaba 1.16).

(I was temporarily very confused, because it turns out that we never gutted vim-selector, so all the references I found were to 'selector#...' rather than 'maktaba#selector#...')

Reply:
Yes, it's breaking, but we never migrated anything to use this library. Moreover, I couldn't find a single external reference to this function, so I believe it's unused.

Reply:
Alrighty, sgtm then. Thanks!

Reply:
Now that you mention it, would be good form to leave a stub `maktaba#selector#Create` behind that throws an error like `Deprecated: call maktaba#ui#selector#Create instead.` until the next minor version bump.

Reply:
I'll send a follow-up